### PR TITLE
CSS Framework CA

### PR DESCRIPTION
**Pull request by eskiltorset**
I accidentally did all my work in sass-setup rather than cs-fw branch. Sorry about that.
inside this branch i have commited my CSS Framework project - with a login page, feed and profile page made by using Bootstrap and SASS.

### CODE REVIEW

File: src/scss/styles.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+.profile {
+    color:  white;
+}
+
+.posts {
+    background-color: $fourth-colour;
+    
+}
+
+.card {
```
Comment (sulenchy) [inline src/scss/styles.scss:49]: There is a class in Bootstrap called card. You can extend it if you like. See the example in the https://content.noroff.dev/css-frameworks/bootstrap-with-sass.html#bootstrap-variables

File: src/scss/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// Colour variables
+$primary-colour: rgb(49, 49, 59);
+$secondary-colour: rgb(199, 98, 98);
```
Comment (sulenchy) [inline src/scss/_variables.scss]: These are your custom variables that have nothing to do with bootstrap. You can simply get an existing variable in Bootstrap and override its value with your new value. See an example

```$blue:    #5d95e8;
$indigo:  #916fc8;
$purple:  #5d3b9c;
$pink:    #ab4578;
$red:     #fa2f44;
$orange:  #ffa053;
$yellow:  #ffdf7e;
$green:   #4fe49e;
$teal:    #9fffe2;
$cyan:    #8eecff;```

**Review by sulenchy — CHANGES_REQUESTED** [review]
@eskiltorset see my comments above

File: src/scss/_variables.scss (hunk 2 above)
Comment (eskiltorset) [reply src/scss/_variables.scss]: Hey, i have resolved it now.

File: src/scss/styles.scss (hunk 1 above)
Comment (eskiltorset) [reply src/scss/styles.scss:49]: Nothing i'd like to extend about the .card class - but now i'll know!

**Pull request merged**